docs/iot-platform-sdk/running-the-demo.md

```diff
@@ -454,47 +454,111 @@ The above directory contains several subdirectories, each representing a device,
 
-These files can be replaced to customize the downloading process. For example, the following JavaScript\* Object Notation (JSON) represents that a file 'package.sh' will be sent to the device, and will be renamed to the value stored in the file 'package\_name', say 'linux64.sh'. Similarly, the file 'payload.bin' will be downloaded to the device and will be renamed to the value stored in the file 'payload\_name', say 'payload.bin'. Finally, the value of the file 'sh-linux64', '/bin/sh linux64.sh', will be executed as command in the device.
+## Setting up ServiceInfo transfer
+The ServiceInfo communication involves the transfer of files from the Owner to the Device.
 
-```
+These files can be a binary, an executable script or even a normal text file. The files that are to be transferred to the device is stored in the following path  `<iot-platform-sdk>/demo/ocs/config/db/v1/values`.
+
+While transferring, the files are encoded in base64 format.  The ServiceInfo configuration file  **_svi.json_** is available in the following path `<iot-platform-sdk>/demo/ocs/config/db/v1/devices/<DEVICE-GUID>`
+
+The structure of **_svi.json_** can be modified to customize the ServiceInfo transfer process. For example, the following JavaScript* Object Notation (JSON) represents that a file **'package.sh'** will be sent to the
+device and will be renamed to the value stored in the file **'package_name'**, say 'linux64.sh'. Similarly, the file **'payload.bin'** will be downloaded to the device and will be renamed to the value stored in the file **'payload_name'**, say 'payload.bin'. Finally, the value of the file **'binsh-linux64'**, **'/bin/sh linux64.sh'**, will be executed as command in the device.
+
+```json
 [
   {
     "module": "sdo_sys",
     "msg": "filedesc",
     "valueLen": -1,
-    "valueId": "package_name",
+    "valueId": "payload_name",
     "enc": "base64"
   },
   {
     "module": "sdo_sys",
     "msg": "write",
     "valueLen": -1,
-    "valueId": "package.sh",
+    "valueId": "payload.bin",
     "enc": "base64"
   },
   {
     "module": "sdo_sys",
     "msg": "filedesc",
     "valueLen": -1,
-    "valueId": "payload_name",
+    "valueId": "package_name",
     "enc": "base64"
   },
   {
     "module": "sdo_sys",
     "msg": "write",
     "valueLen": -1,
-    "valueId": "payload.bin",
+    "valueId": "package.sh",
     "enc": "base64"
   },
   {
     "module": "sdo_sys",
     "msg": "exec",
     "valueLen": -1,
-    "valueId": "sh-linux64",
+    "valueId": "binsh-linux64",
     "enc": "base64"
   }
 ]
 ```
+
+!!! Note
+     The ordering of message blocks are important for **Client-SDK** / **Client-CSDK TPM** device. The “filedesc” block must always be followed by “write” block and if any executable script is transfered then the “exec” block must be immediately followed after both “filedesc” and “write” blocks.
+
+For the scenario of transferring multiple executables to the Device, then the following order of blocks should be adhered.
```

Comment:
Suggest rewording the statement a bit:
For the scenario of transferring multiple executables to the Device, following order of blocks should be maintained.

Reply:
Done